src/utils/getDisplayName.ts

```diff
@@ -20,3 +20,26 @@ export const getDisplayNameWithFallback = function(displayName: string, source:
 	// - in other cases: should not happen, but can not be empty either
 	return t('spreed', 'Deleted user')
 }
+
+export const getDisplayNamesList = function(displayNames: string[], maxLength: number): string {
```

Comment:
```suggestion
export function getDisplayNamesList(displayNames: string[], maxLength: number): string {
```

And then rules for functions work